ggml-metal.m

```diff
@@ -210,6 +210,10 @@ static void ggml_metal_log(enum ggml_log_level level, const char* format, ...){
             GGML_METAL_LOG_INFO("%s: default.metallib not found, loading from source\n", __func__);
 
             NSString * sourcePath = [bundle pathForResource:@"ggml-metal" ofType:@"metal"];
+            if(sourcePath == nil) {
```

Comment:
```suggestion
            if (sourcePath == nil) {
```

Reply:
Oops, yeah, my bad :) Thanks for fixing!